src/coreclr/scripts/jitrollingbuild.py

```diff
@@ -107,6 +113,11 @@
 download_parser.add_argument("-target_dir", help=target_dir_help)
 download_parser.add_argument("--skip_cleanup", action="store_true", help=skip_cleanup_help)
 
+# subparser for updatejitmap
+updatejitmap_parser = subparsers.add_parser("updatejitmap", description=updatejitmap_description, parents=[common_parser])
+
+updatejitmap_parser.add_argument("-az_storage_key", help="Key for the clrjit Azure Storage location. Default: use the value of the CLRJIT_AZ_KEY environment variable.")
```

Comment:
Perhaps make this a common key so both `upload` and `updatejitmap` can share? Then you can remove the new code you added specifically for `updatejitmap`?